packages/styles/scss/components/structured-list/_structured-list.scss

```diff
@@ -293,8 +293,16 @@
 
   .#{$prefix}--structured-list-cell[icon],
   :host(#{$c4d-prefix}-structured-list-cell[icon]) {
-    &[icon='checkmark'] {
+    &[icon='checkmark'] svg {
       color: $support-success;
     }
+
+    svg {
+      margin-inline-end: $spacing-03;
+    }
+  }
+
+  .#{$prefix}--structured-list-cell-icon-text-container {
+    display: flex;
```

Comment:
With this being `display: flex`, the `svg` flex item needs `flex-shrink: 0` to that it maintains its original size. As it is, if you have multiple lines of text for `icon-text`, the icon shrinks:

![image](https://github.com/user-attachments/assets/53c580b8-de2a-4e58-9ef7-a15b796eb95f)

Reply:
Should be fixed now:

<img width="773" alt="Screenshot 2024-11-12 at 10 26 02 AM" src="https://github.com/user-attachments/assets/b383a539-a874-48ea-bf2f-fc705914e20b">